ts-fold.el

```diff
@@ -342,15 +344,17 @@ This function is borrowed from `tree-sitter-node-at-point'."
 (defun ts-fold--format-overlay-text (beg end)
   "Return the text to display in the overlay for the fold from BEG to END."
   (let ((summary (and ts-fold-summary-show
-                      (ts-fold-summary--get (buffer-substring beg end))))
-        (line-count (and ts-fold-line-count-show (count-lines beg end))))
+                      (ts-fold-summary--get (buffer-substring beg end)))))
     (cond
-     ((and summary line-count)
-      (format "%s %s" summary (format ts-fold-line-count-format line-count)))
-     ((and summary (not line-count))
-      (format "%s %s" summary (ts-fold--truncate-string-ellipsis)))
```

Comment:
I have removed the logic here since the summary handles truncation independently.